frontend/src/container/CreateAlertRule/SelectAlertType/styles.ts

```diff
@@ -0,0 +1,22 @@
+import { Card, Row } from 'antd';
+import styled from 'styled-components';
+
+export const SelectTypeContainer = styled.div`
+	&&& {
+		padding: 1rem;
+	}
+`;
+
+export const AlertTypeCards = styled(Row)`
```

Comment:
there is a prop for same wrap={false} in Row ?

Reply:
This section is still under designed. I prefer adding domains specific styles to make the code more readable (you can easily map ui to the code sections) and allow future changes.  do you see any side effect with this approach?